packages/captp/test/test-disco.js

```diff
@@ -1,5 +1,3 @@
-/* global harden */
-
 import '@agoric/install-ses';
```

Comment:
oh, this tests disconnect? For some reason I thought "disco" was short for "discovery".

Reply:
Remnants of my days in the telecom industry.